import_export/resources.py

```diff
@@ -780,7 +780,7 @@ def import_row(self, row, instance_loader, **kwargs):
             # There is no point logging a transaction error for each row
             # when only the original error is likely to be relevant
             if not isinstance(e, TransactionManagementError):
-                logger.debug(e, exc_info=e)
+                logger.warning(e, exc_info=True)
```

Comment:
Im not opposed to the `exc_info=True` 👍 

However, instead of changing from debug to warning, couldnt you lower the log level in your logging configuration to see these?

Reply:
Yeah indeed, i usually test with logger set to "warning" to remove unnecessary noise. But i find this one to be quite important if it fails. But yeah if these things get logged in production with millions of lines failing then you end up with huge log files and maybe even bigger issues like disk space issues. So i think it's best to keep it to debug.

Reply:
Moved the level to debug again

Reply:
Thanks! LGTM!